src/components/layout/AppHeader.vue

```diff
@@ -53,6 +60,18 @@ const headerMenu = [
 	{ ids: [ 'faq-page' ], localeId: 'faq', url: `/faq?${ campaignParams }` },
 ];
 
+const checkMenuIsUnfocused = () => {
```

Comment:
The function name is confusing to me, especially with the timeout. Is it `onBlur` or `hideNavigationListOnBlur`?

I think the timeout also needs an explanation/comment. From reading the code, I'd guess you put it in because by the time the `blur` event fires, `document.activeElement` has not changed yet. Is that the case?
Also, `setTimeout` without a delay makes me go "huh?" and look up the behavior in  
https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#delay That makes this more clear, because a missing delay means it'll be executed "on the next event cycle", i.e. when the browser is done handling the `blur` event, which supports my guess of what the timeout is for.

Im wondering if a slight delay would be beneficial to users who accidentally moved out of this and then wnat to go back.